src/pages/viewer-app/viewer-app.tsx

```diff
@@ -134,6 +147,9 @@ export const ViewerApp = () => {
   const filtersByAttribute = useSelector((state: RootState) =>
     selectFiltersByAttribute(state)
   );
+  const [showWrongBookmarkWarning, setShowWrongBookmarkWarning] = useState<
+    PageId | false
+  >(false);
```

Comment:
`PageId | false` is confusing because the state name looks like a boolean.
```suggestion
  const [wrongBookmarkPageId, setWrongBookmarkPageId] = useState<
    PageId | null
  >(null);
```

Reply:
Done.